lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -89,155 +93,262 @@ class StorageEnclosureConfig(Command):
         }
     }
 
+    def __init__(self):
+        super().__init__()
+        self.node_id = local_minion_id()
+        self.enc_num = "enclosure-" + ((self.node_id).split('-'))[1]
+        self.flag = False
+
+        if enc_file_path.exists():
+            if Path(enc_file_path).stat().st_size != 0:
+                self.flag = True
+                with open(enc_file_path, "r") as file:
+                    self.enclosure_id = file.read().replace('\n', '')
+        else:
+            self.enclosure_id = None
+
+    def store_enclosure_id(self):
```

Comment:
If possible can we use concept of mapping to avoid writing similar code for store functions.

refer this: https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/lr-cli/cortx_setup/commands/node/prepare/config.py#L34